src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.targets

```diff
@@ -86,6 +86,8 @@
 
   <Import Project="$(MSBuildThisFileDirectory)Packaging.targets" Condition="'$(ExcludePackagingImport)'!='true' AND '$(MSBuildProjectExtension)' == '.pkgproj'" />
 
+  <Import Project="$(MSBuildThisFileDirectory)codeAnalysis.props" />
```

Comment:
Given this is a props file, wouldn't it make more sense to include it in Build.Common.props?